upgrade/upgrade-scripts/upgrade

```diff
@@ -208,6 +251,10 @@ not-in-place)
 	verify_upgrade_is_allowed
 	upgrade_not_in_place "$@"
 	;;
+rollback)
+	shift 1
+	rollback "$@"
```

Comment:
Here and elsewhere, we aren't expecting any arguments, so we shouldn't be passing `"$@"` but rather should fail if any extra arguments are passed.

Reply:
Yea, we could fail. I didn't do that because the other commands didn't handle it that way. I'll open a bug to follow up on this.

Reply:
I've opened: https://github.com/delphix/appliance-build/issues/220